docs/blog/2019-04-03-how-we-think-about-product-at-gatsby/index.md

```diff
@@ -0,0 +1,130 @@
+We typically think of MVPs and Minimum Viable Products, but that often means that we are creating something that’s very well thought out and crafted _before_ we check to make sure it’s something people want or something we should spend our time doing. By focusing on research, discovery, and _prototyping_ we keep the focus on our end users and retain the ability to rapidly test potential solutions.
+
+### How we focus on personas
+
+Because we want to stay user-focused, we will also be doing discovery and usability research against specific personas that are important to Gatsby. As we learn more about our audience, we’ll get a better understanding of current personas and start to see new personas that we should define.
+How we prioritize
+Since we’ll be doing a lot of discovery, usability research, and prototyping, that means we’ll have quite a bit of prioritization to do.
+
+We’ll prioritize based on OKRs we talked about before. Objectives show what we want to accomplish and key results show how we’ll measure that we’ve accomplished it.
+
+The business vision and objective is set at the company level, and the business results are set by the individual teams. This means that teams will have the autonomy to hit the business objective in the best way they see fit.
+How we put it all together
+Outcome-based roadmaps
+
+Once we understand the ideas we want to prioritize, the objectives we’re trying to hit, and the results we’ll use to measure those outcomes, we can put together a roadmap that shows where we want to focus.
+
+Typical roadmaps are difficult to use because they don’t communicate information in a way that addresses everyone’s needs.
+
+To make sure that we don’t fall into this trap, we will do outcome-based roadmaps. Our roadmap will include:
+our business vision
+our objectives
+our business results
+
+This will help us focus our roadmaps on business problems we’re trying to solve, rather than a general feature that we’re not sure is important.
+
+Where will all of this information live?
+This is obviously a lot of information to gather and we want to be sure we capture the important parts and provide a reference and source of truth for the team to work from.
+Product Requirement Documents
+Specifically for product, we create Product Requirement Documents (PRDs) that will explain what we want to do and why. They will include information that we need to scope out the product or feature, and also include the evidence that shows why we should do this. Because we want to make sure our decisions are based on actual problems, research, and data we always include that in the PRDs as justification.
+RFCs
```

Comment:
```suggestion

RFCs

```

```suggestion

### RFCs

```

Reply:
It'd be cool to link to RFCs and PRD examples here, especially if we have current ones~

Reply:
I don't think we have any PRDs on gatsbyjs/gatsby right now that are current, but I like this idea and is something we should chat about.